modules/settings/settings.php

```diff
@@ -260,10 +222,12 @@ public function helper_settings_validate_and_save() {
 			return false;
 		}
 
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- wp_verify_nonce() is already doing the necessary work
 		if ( ! current_user_can( 'manage_options' ) || ! wp_verify_nonce( $_POST['_wpnonce'], VIP_Workflow::instance()->$module_name->module->options_group_name . '-options' ) ) {
 			wp_die( esc_html__( 'Cheatin&#8217; uh?' ) );
 		}
 
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- check is already done above
```

Comment:
This does look pretty dangerous at first glance, as we're getting an array from `$_POST` directly using a dynamic property name, and then and it's unclear that validation is actually being called due to the dynamic module setup:

https://github.com/Automattic/vip-workflow-plugin/blob/ceebcba6079ee00e4bbf1a37849f7b2c848d79c2/modules/settings/settings.php#L233-L236

For right now, can we remove the `if` wrapper on the code below this to make it obvious that we're calling `settings_validate()` on these values we get from `$_POST`?

Reply:
Yip, we don't need that if check anymore so makes sense to kill it entirely